scipy/spatial/qhull.pyx

```diff
@@ -2150,6 +2150,20 @@ cdef int _get_delaunay_info(DelaunayInfo_t *info,
     return 0
 
 
+def _has_nan(points):
+    """
+    has_nan(points)
+
+    Checks whether a point in an array of points has a NaN value.
+
+    .. versionadded:: 0.17.0
```

Comment:
To a potential reviewer: I was not sure of how extensively I should document a private function, so I went this far. Any feedback about this is appreciated.